distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -235,6 +227,10 @@ type File
     with_output_stream open_options action =
         Resource.bracket (this.new_output_stream open_options) (_.close) action
 
+    output_stream options = @Builtin_Method "File.output_stream"
+
```

Comment:
Will these two methods return `java.io.*putStream` wrapped as host `TruffleObject` and let any Enso user call any method on it?

Reply:
Yes, since *_Stream is not a builtin type I do some inference to make it happen. This refers to https://github.com/enso-org/enso/pull/3471#discussion_r885066994